docs/_ug/Installation.md

```diff
@@ -5,7 +5,7 @@
 If you haven't installed FoodRem, simply follow these steps to set it up:
 
 1. Ensure you have [Java 11](https://docs.oracle.com/en/java/javase/11/install/overview-jdk-installation.html) installed on your computer.
-1. Download the latest `foodrem.jar` file from our [release page](https://github.com/AY2223S1-CS2103T-W16-2/tp/releases/)
+1. Download the latest `foodrem.jar` file from our [release page.](https://github.com/AY2223S1-CS2103T-W16-2/tp/releases/)
 1. Copy the jar file to an empty folder. This will be your home folder for FoodRem.
 1. Double-click on the jar file to launch FoodRem.
```

Comment:
Actually, since these are list points, would it make more sense if everything was standardised to no full stops?

Reply:
Yes, will do!